src/containers/treePage/ducks/types.ts

```diff
@@ -289,6 +290,34 @@ export interface AdoptedSites {
   adoptedSites: number[];
 }
 
+export interface SiteEntryImage {
+  imageId: number;
+  uploaderUsername: string | null;
+  uploadedAt: string;
+  imageUrl: string;
+}
```

Comment:
⛏️ IIRC usernames are non-null, so I think we can type `uploaderUsername` as just `string`